ipld/plugin/nmt.go

```diff
@@ -351,7 +315,12 @@ func MustCidFromNamespacedSha256(hash []byte) cid.Cid {
 	return cidFromHash
 }
 
+// cidPrefixSize is the size of the prepended buffer of the CID encoding
+// for NamespacedSha256. For more information, see:
+// https://multiformats.io/multihash/#the-multihash-format
+const cidPrefixSize = 4
```

Comment:
Hm, they don't have an exported const we can use? :(

Reply:
AFAIK, no